extensions/target-specific/chromium/chromium-helpers/chromium-helpers.js

```diff
@@ -82,7 +82,7 @@ var Chromium = null;
             _help_BrowserProcessEquivalentModules: {
                 description: "List of modules used in the Chromium browser process.",
             },
-            BrowserProcessEquivalentModules: ["msedge", "accessibility", "chrome"],
+            BrowserProcessEquivalentModules: ["msedge", "accessibility"],
```

Comment:
"msedge" is basically a rename of "chrome", so we should move that over as well.

Reply:
Done